kubesae/info.py

```diff
@@ -1,24 +1,45 @@
 import invoke
 import yaml
 
-@invoke.task(default=True)
-def get_ansible_vars(c, var=None):
-    """Inspect ansible variables
+@invoke.task("print_ansible_vars")
```

Comment:
I like the name change! I think there's a reference in the README that needs updating.